.github/workflows/recurring_issues.yml

```
@@ -0,0 +1,31 @@
+    - cron: "0 2 1 * *"
+
+jobs:
+  create_issue:
+    name: Create Monthly Maintenance Audit Issue
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%m-%Y')"
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)